src/Features/Core/Portable/ConvertTupleToStruct/AbstractConvertTupleToStructCodeRefactoringProvider.cs

```diff
@@ -872,6 +878,9 @@ private static SyntaxNodeOrToken ConvertArgumentOrToken(SyntaxGenerator generato
             return constructor;
         }
 
+        private static string GetConstructorParameterName(string name)
+            => name.ToCamelCase(trimLeadingTypePrefix: false);
```

Comment:
note: technically this is not accurate.  while this is the norm (and def better than where we started from), this doesn't necessarily match the user's actual stated preference for their parameters.  i.e. a user could have a different naming style.  consider updating this to respect that style.

note: this is a not a blocker on the PR.  given that camel-case is likely correct for 95%+ of our users (and we were broken before anyways), it's totally fine to just get to this point and have us wait to see if anyone ever cares about the niche case of having a different parameter style *and* using this feature.

That said, it's likely worth a comment in the code that the above line is for expediency with teh common case, and not for correctness.

Reply:
I agree this isn't a blocker, which is why I left the existing behavior, but I definitely want to follow up on this, as its a pet peeve of mine when our code actions don't honour my settings. I just don't know how to do anything about it yet :)